FCCee/compact/CLD_ARC_o1_v1/README.md

```diff
@@ -0,0 +1,25 @@
+ARC detector V0
+=======================
+
+Array of RICH Cells (ARC) is a novel RICH detector concept. The ARC detector is made up by two parts, a barrel and two endcaps. The barrel consist in the repetition of 16 unique cells, 27 times around phi, and mirrored over z axis. Each endcap consist in the repetition of 6 times of a sector. One sector is made of 21 unique cells, some of the reflected within the sector. Each unique cell
+* is filled by C4F10 radiator gas
+* has a spherical mirror in the outer radial part of the cell
+* and a sandwich of three layers in the inner radial part, from inner to outer, a cooling plate, a light sensor, and an aerogel radiator.
+
+Each cell has 5 characteristic parameters, which correspond to
+* the position of the mirror (2),
+* the position and angle of the sensor (3)
+* optional parameter (`xxx_DetZOffset`) can be given to increase the distance of the sensor from the vessel, so the volumes do not overlap
+There parameters are defined as `DD4hep` constants in the `RadiatorCell_FinalOptimization.xml` file.
+
+The original design included few more cells, 2 more on the edge of the barrel, and some more in the endcap. These cells are not included in the current design because they require some extra work to be properly implemented.
+
+It has a radial depth of 20 cm, outer radius of 2.1 m and a length of 4.4 m. Such dimensions are hardcoded now. In a later version, global dimensions and some other internal parameters will be setup freely in the compact file.
```

Comment:
Can you maybe add here (or in the other README that will go in detector/PID) that if the radius is changed, all the free parameters have to be re-optimized?